manifests/4.3/cluster-logging.v4.3.0.clusterserviceversion.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
     createdAt: 2018-08-01T08:00:00Z
     support: AOS Logging
     # The version value is substituted by the ART pipeline
-    olm.skipRange: ">=4.1.0 <4.2.0"
+    olm.skipRange: ">=4.1.0 <4.3.0"
```

Comment:
i really meant your test code should hack this, not that you should check it in this way, but i guess it's fine as long as you change it before 4.3 ships.  (or you're ok w/ users upgrading from 4.1 directly to 4.3)

Reply:
This is why I logged #248.  I don't think we should but until we can get a cluster that has OLM content that is appropriate for a given release we'll have to do this